File: dailyHOTEL/app/src/main/java/com/daily/dailyhotel/screen/copy/java/CopyView.java
package com.daily.dailyhotel.screen.copy.java;

import com.daily.base.BaseActivity;
import com.daily.base.BaseDialogView;
import com.twoheart.dailyhotel.databinding.ActivityCopyDataBinding;

public class CopyView extends BaseDialogView<CopyInterface.OnEventListener, ActivityCopyDataBinding> implements CopyInterface.ViewInterface
{
    public CopyView(BaseActivity baseActivity, CopyInterface.OnEventListener listener)
    {
        super(baseActivity, listener);
    }

    @Override
    protected void setContentView(final ActivityCopyDataBinding viewDataBinding)
    {
        if (viewDataBinding == null)
        {
            return;
        }

        initToolbar(viewDataBinding);
    }

    @Override
    public void setToolbarTitle(String title)
    {
        if (getViewDataBinding() == null)
        {
            return;
        }

        getViewDataBinding().toolbarView.setTitleText(title);
    }

    private void initToolbar(ActivityCopyDataBinding viewDataBinding)
    {
        if (viewDataBinding == null)
        {
            return;
        }

        viewDataBinding.toolbarView.setOnBackClickListener(v -> getEventListener().onBackClick());
    }
}

File: dailyHOTEL/app/src/main/java/com/daily/dailyhotel/screen/copy/kotlin/CopyView.kt
package com.daily.dailyhotel.screen.copy.kotlin

import com.daily.base.BaseDialogView
import com.twoheart.dailyhotel.databinding.ActivityCopyDataBinding

class CopyView(activity: CopyActivity, listener: CopyInterface.OnEventListener)//
    : BaseDialogView<CopyInterface.OnEventListener, ActivityCopyDataBinding>(activity, listener), CopyInterface.ViewInterface {

    override fun setContentView(viewDataBinding: ActivityCopyDataBinding) {
        initToolbar(viewDataBinding)
    }

    override fun setToolbarTitle(title: String?) {
        viewDataBinding.toolbarView.setTitleText(title)
    }

    private fun initToolbar(viewDataBinding: ActivityCopyDataBinding) {
        viewDataBinding.toolbarView.setOnBackClickListener { eventListener.onBackClick() };
    }
}